packages/v4/src/content/developer-resources/accessibility-testing.md

```diff
@@ -0,0 +1,106 @@
+
+
+<br />
+
+### Checklist for testing
+These are some of the main areas we check for on the PatternFly team. We recommend that you check for these same areas in your applications, too. 
+
+**General questions**
+
+<input type="checkbox" unchecked /> VO Rotor Navigation can discover all information <br/>
+<input type="checkbox" unchecked /> VO Shortcut Navigation can discover all information <br/>
+<input type="checkbox" unchecked /> VO Cursor Navigation can discover all information. (dependent on component and use case- easy way to navigate to component) <br/>
+<input type="checkbox" unchecked /> Tabbed Keyboard Navigation can discover all information
+- However, if content should be hidden from a screen reader, for instance, if it’s offscreen or just presentational, make sure that content is set to aria-hidden=”true”.  <br/>
+
+<input type="checkbox" unchecked /> Is it understandable and usable? <br/>
+  - <input type="checkbox" unchecked /> When you navigate to an item by keyboard or by screen reader, you can easily understand and use the item <br/>
+  - <input type="checkbox" unchecked /> Flow of information makes sense when navigating - screen readers navigate the page in DOM order, if you’ve used CSS to visually reposition elements, they may be announced in a nonsensical sequence. If you need something to appear earlier in the page, try to physically move it earlier in the DOM.
+
+**Accessibility points** <br />
+<input type="checkbox" unchecked /> **Structure:** Need to map the visual information architecture to the various rotor menus that exist by default. <br />
+- <input type="checkbox" unchecked /> **Check rotor:** are there good headings, landmarks, links, form controls, etc? Do they make sense, are descriptive but not overloading with info? <br />
+  - <input type="checkbox" unchecked /> Headings (check that heading levels convey structure/content and do not skip)
+    - A common practice is to use a single h1 for the primary headline or logo on a page, h2s to designate major sections, and h3's in supporting subsections <br />
+  - <input type="checkbox" unchecked /> Links <br />
+  - <input type="checkbox" unchecked /> Form Controls <br />
+  - <input type="checkbox" unchecked /> Landmarks <br />
+  - <input type="checkbox" unchecked /> Tables <br />
+
+
+<input type="checkbox" unchecked /> **Labels:**
+- <input type="checkbox" unchecked /> **Link** labels should be unique unless they have the same url, should be descriptive/informative
+- <input type="checkbox" unchecked /> **Buttons, form controls**, etc.
+  - Go through form controls and check if they all have labels (accessible vs. good experience)
+  - **Expandable buttons** should have some indication that it has the expandable control
+- <input type="checkbox" unchecked /> **Form input** has to have some label even if not visible
+- <input type="checkbox" unchecked /> **Icons**: where you don’t visibly see text there should still be some kind of text for screen reader
```

Comment:
I recommend formatting each item consistently. Some are sentences, some aren't sentences, and some start with a bolded word and colon (:) whereas others don't. I don't think it matters which format we go with, as long as they're all consistent for clarity and easy reading/skimming 👍